core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -182,12 +174,40 @@ private void createSchemaTopic() throws StoreInitializationException {
     schemaTopicProps.put(LogConfig.CleanupPolicyProp(), "compact");
 
     try {
-      AdminUtils.createTopic(zkUtils, topic, 1, schemaTopicReplicationFactor, schemaTopicProps);
+      AdminUtils.createTopic(zkUtils, topic, 1, schemaTopicReplicationFactor, schemaTopicProps,
+              RackAwareMode.Enforced$.MODULE$);
     } catch (TopicExistsException e) {
       // This is ok.
     }
   }
 
+  private String getBrokerEndpointsFromZk(Seq<Broker> brokers) {
```

Comment:
I'm happy to write unit tests. Should I make the method `public` or `protected` to facilitate this? What's the best practice here?